buildSrc/src/main/resources/index.html

```diff
@@ -378,70 +315,11 @@
             quotes: none;
         }
     </style>
-
```

Comment:
We are still using Google Analytics

Reply:
Added back https://github.com/micronaut-projects/micronaut-docs-index/pull/3/commits/d8bf5c601ea12c16a8f39d62e86fef70902b30ff